code/edit.py

```diff
@@ -36,3 +36,8 @@ def words_right(n: int):
         """Moves right by n words."""
         for _ in range(n):
             actions.edit.word_right()
+
+    def insert_blank_line_below():
```

Comment:
Is this not the same as [`edit.line_insert_down`](https://github.com/knausj85/knausj_talon/blob/dded1d6e83c9cedad12ce7928e5c317b25f8306f/code/platforms/mac/edit.py#L87-L88)?